backend/lettercraft/settings.py

```diff
@@ -120,8 +120,6 @@
 
 USE_I18N = True
 
-USE_L10N = True
```

Comment:
This setting produces warnings in `pytest` as it is deprecated since Django 4.0 (we are running 4.2). `True` is the default setting anyway, so I think we can safely remove it.